solr/solr-ref-guide/modules/deployment-guide/pages/backup-restore.adoc

```diff
@@ -234,38 +234,43 @@ http://localhost:8983/solr/gettingstarted/replication?command=restorestatus&wt=x
 The status value can be "In Progress", "success" or "failed".
 If it failed then an "exception" will also be sent in the response.
 
-=== Create Snapshot API
+[[create-snapshot-api]]
+== CREATE: Create a Snapshot
 
 The snapshot functionality is different from the backup functionality as the index files aren't copied anywhere.
 The index files are snapshotted in the same index directory and can be referenced while taking backups.
 
 You can trigger a snapshot command with an HTTP command like this (replace "techproducts" with the name of the core you are working with):
 
-.Create Snapshot API Example
-[source,text]
+[.dynamic-tabs]
+--
+[example.tab-pane#v1createsnapshot]
+====
+[.tab-label]*V1 API*
+
+[source,bash]
 ----
-http://localhost:8983/solr/admin/cores?action=CREATESNAPSHOT&core=techproducts&commitName=commit1
+curl -X POST http://localhost:8983/solr/admin/cores?action=CREATESNAPSHOT&core=techproducts&commitName=commit1
+
 ----
+====
 
-The `CREATESNAPSHOT` request parameters are:
+[example.tab-pane#v2createsnapshot]
+====
+[.tab-label]*V2 API*
 
-`commitName`::
-+
-[%autowidth,frame=none]
-|===
-|Optional |Default: none
-|===
-+
-The name to store the snapshot as.
+With the v2 API, the core and snapshot names are part of the path, no longer query parameters.
```

Comment:
[+1] Awesome docs, thanks for adding these!  Big fan of the use of "tabs" in the docs to handle v1/v2 examples.

Only nitpick I'd make here is to drop or rephrase the "no longer query parameters" bit (maybe use "instead of"?).  But even that's probably personal preference.

Reply:
No problem! I actually just copied the structure from elsewhere in the docs, so it wasn't much work on my part. I did go ahead and change the wording since I thought the "instead of" read better.